donor-portal/src/components/navbar/index.tsx

```diff
@@ -22,32 +22,33 @@ export default function NavBar() {
         <div className="logo" />
       </Link>
       <div className="nav-links">
-        <Link to="/about-us" className="nav-link">
+        <NavLink to="/" className="nav-link" exact>
+          Home
+        </NavLink>
+        <NavLink to="/about-us" className="nav-link">
           About
-        </Link>
-        <span>|</span>
-        <Link to="/donors" className="nav-link">
+        </NavLink>
+        <NavLink to="/donors" className="nav-link">
           Donors
-        </Link>
-        <span>|</span>
-        <Link to="/suppliers" className="nav-link">
-          Hospital and Suppliers
-        </Link>
-        <span>|</span>
+        </NavLink>
+        <NavLink to="/suppliers" className="nav-link">
+          Hospital & Suppliers
+        </NavLink>
         <Link to="/news-room" className="nav-link">
```

Comment:
Should we convert this as well?

Reply:
Navlink automatically provides an active class. Link doesn't.

Reply:
Yh 4got about tht.. thnx